src/views/Project/index.jsx

```diff
@@ -36,6 +41,31 @@ const tagReposMapping = repositories =>
     },
   }),
 })
+@withStyles(theme => ({
+  root: {
+    background: theme.palette.background.default,
+  },
+  header: {
+    height: 60,
+    paddingBottom: theme.spacing.unit,
+  },
+  container: {
+    marginTop: 60,
+    padding: 2 * theme.spacing.unit,
+    minHeight: `calc(100vh - 180px - ${3 * theme.spacing.unit}px)`,
+  },
+  link: {
+    textDecoration: 'none',
+    position: 'absolute',
+    alignItems: 'center',
+    '& svg': {
+      fill: theme.palette.common.white,
+    },
+  },
+  title: {
+    marginLeft: 41,
```

Comment:
Replace `marginLeft: 41` with ` padding: '0 41px' `. That's another way of making sure the title stays in the center.

Reply:
@fiennyangeln Let's give that a try and see if it works for you.